Mage.Tests/src/test/java/org/mage/test/cards/triggers/NeeraWildMageTest.java

```diff
@@ -0,0 +1,52 @@
+package org.mage.test.cards.triggers;
+
+import mage.constants.PhaseStep;
+import mage.constants.Zone;
+import org.junit.Test;
+import org.mage.test.serverside.base.CardTestPlayerBase;
+
+/**
+ *
+ * @author rjayz
+ */
+public class NeeraWildMageTest extends CardTestPlayerBase {
```

Comment:
Please move this test under the folder for other clb cards. cards/single/clb (or something very similar)

Reply:
Thanks, I have moved the test